src/less/dialog/base/dialog.less

```diff
@@ -186,12 +188,63 @@
     }
 }
 
+// Mini dialog (A.K.A Dialog Infotip) variant
+.dialog-infotip[role="dialog"] {
+    .dialog-base();
+}
+
+.dialog-infotip__window {
+    .dialog-window(400px);
+
+    flex-direction: row-reverse;
+    left: 0;
+    min-width: 200px;
+    padding-bottom: 15vh;
+    top: 50%;
+    transform: translateY(-25%);
+
+    &::before {
+        border-radius: 7px;
+    }
+}
+
+.dialog-infotip__header {
+    .dialog-header-content(0);
+
+    align-items: flex-start;
+}
+
+.dialog-infotip__close {
+    background-color: @dialog-back-close-background-color;
+    border: 0;
+    color: @dialog-back-close-color;
+    cursor: pointer;
```

Comment:
Nitpick, but please remove `cursor: pointer`. For a11y reasons we only have pointer on links, not buttons. Also I don't think `z-index` is needed...?

Reply:
ACK on `cursor`

`z-index` is something that is copied over from `dialog__close`. Not sure why it was needed back during dialog impl. Could be to give precedence (Top of) over default elements just in case.